chef/cookbooks/cinder/recipes/volume.rb

```diff
@@ -43,6 +43,8 @@ def make_loopback_file(node, volume)
   max_fsize = ((`df -Pk #{encl_dir}`.split("\n")[1].split(" ")[3].to_i * 1024) * 0.90).to_i rescue 0
   fsize = max_fsize if fsize > max_fsize
 
+  FileUtils.mkdir_p(fdir) unless File.exists?(fdir)
```

Comment:
This will not conflict with packages 
```
directory fdir do
  owner node[:cinder][:user]
  group node[:cinder][:group]
  mode '0755'
  action :create
  not_if { File.directory?(fdir) }
end
```